src/misc/setup.cpp

```diff
@@ -981,6 +981,24 @@ void Config::StartUp()
 	(*_start_function)();
 }
 
+Verbosity Config::GetStartupVerbosity() const
+{
+	const Section* s = GetSection("dosbox");
+	assert(s);
+	const std::string user_choice = s->GetPropValue("startup_verbosity");
+
+	if (user_choice == "high")
+		return Verbosity::High;
+	if (user_choice == "medium")
+		return Verbosity::Medium;
+	if (user_choice == "low")
+		return Verbosity::Low;
+	if (user_choice == "quiet")
+		return Verbosity::Quiet;
+	// auto-mode
+	return (cmdline->HasDirectory() || cmdline->HasExecutable() ? Verbosity::Low : Verbosity::High);
```

Comment:
Maybe this line is long enough now to warrant a form:
```suggestion
	if (cmdline->HasDirectory() || cmdline->HasExecutable())
		return Verbosity::Low;
	else
		return Verbosity::High;
```

Reply:
Looks better!